<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addSongWindow</class>
 <widget class="QMainWindow" name="addSongWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Song</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="song_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>310</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter song title</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_couplet_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add couplet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_chour_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add chour</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_bridge_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add bridge</string>
    </property>
   </widget>
   <widget class="QListWidget" name="song_list">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>401</width>
      <height>521</height>
     </rect>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="uniformItemSizes">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_song_btn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add song</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_item_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>remove item</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
